Directory.Packages.props

```diff
@@ -1,26 +1,31 @@
-<Project>
-
-  <PropertyGroup>
-    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    <!-- Using multiple feeds isn't supported by Maestro: https://github.com/dotnet/arcade/issues/14155. -->
-    <NoWarn>$(NoWarn);NU1507</NoWarn>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageVersion Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotNetRemoteExecutorVersion)" />
-    <PackageVersion Include="Microsoft.DotNet.XUnitExtensions" Version="$(MicrosoftDotNetXUnitExtensionsVersion)" />
-    <PackageVersion Include="Microsoft.DotNet.Build.Tasks.TargetFramework" Version="$(MicrosoftDotNetBuildTasksTargetFrameworkVersion)" />
-  </ItemGroup>
-
-  <!-- These versions are not updated by dependency flow as they aren't produced live anymore. -->
-  <ItemGroup>
-    <PackageVersion Include="System.Reflection.Emit" Version="4.7.0" />
-    <PackageVersion Include="System.ValueTuple" Version="4.5.0" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <GlobalPackageReference Include="Microsoft.DotNet.ApiCompat.Task" Version="9.0.100-preview.2.24102.10" />
-  </ItemGroup>
-
-</Project>
+<Project>

```

Comment:
Many of the line changes in this file don't look right. Maybe SBOM related? Please undo them.